### ISSUE TITLE
Not working for node v0.10

### ISSUE THREAD

**Issue by cyclops24** [chill117/proxy-lists#15]
Hi,
I try use your package as below but I got error:
```
var ProxyLists = require('proxy-lists');
 
var options = {
	anonymityLevels: ['anonymous', 'elite']
};
 
// `gettingProxies` is an event emitter object. 
var gettingProxies = ProxyLists.getProxies(options);
 
gettingProxies.on('data', function(proxies) {
	// Received some proxies. 
	console.log(proxies);
});
 
gettingProxies.on('error', function(error) {
	// Some error has occurred. 
	console.error(error);
});
 
gettingProxies.once('end', function() {
	// Done getting proxies. 
});
```

And this is my error message:

```
/media/username/projects/ProxyFetcher/node_modules/proxy-lists/index.js:70
		var emitter = new EventEmitter();
		              ^
TypeError: object is not a function
    at Object.module.exports.getProxies (/media/username/projects/ProxyFetcher/node_modules/proxy-lists/index.js:70:17)
    at Object.<anonymous> (/media/username/projects/ProxyFetcher/proxyFetcher.js:8:33)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
```
where is my mistake?

**Comment by chill117** [comment]
What version of node are you running?

**Comment by cyclops24** [comment]
Hi @chill117 ,
My node version is v0.10.37.

**Comment by chill117** [comment]
All fixed. The latest release (1.5.2) has the fixes for node 0.10

**Comment by cyclops24** [comment]
Thanks @chill117 it's working now.
